doc/doxygen/src/ForHosts.dox

```diff
@@ -131,6 +136,14 @@
  *
  * @subsection host_todo_required_publishing Required for Publishing
  *
+ * - Ask the manager if it supports the publishing workflow by calling
+ *   @fqref{hostApi.Manager.hasCapability} "Manager.hasCapability",
+ *   passing @fqref{hostApi.Manager.Capability.kPublishing}
+ *   "kPublishing", along with any other capabilities required during
+ *   the publishing procedure (e.g. you may need to
+ *   @fqref{hostApi.Manager.Capability.kResolution} "resolve"
+ *   information during publishing.
```

Comment:
[Fix] Resolving during publishing is predicated on the management policy imbuing the `ResolvesFutureEntitiesTrait`, so unless you absolutely require write resolves then you don't want to disable all publishing without it. As noted above too, that check is technically redundant too.

Reply:
Yeah, this makes things a fair bit simpler, just check for publishing up front when doing publishing. https://github.com/OpenAssetIO/OpenAssetIO/pull/1141/commits/8798160fb07a0ee76a72d751d66ba177c9f6abec